.github/workflows/trunk-artifacts.yml

```diff
@@ -0,0 +1,115 @@
+# Copyright 2025, Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+# These jobs build ISPC for trunk on every push to main pushing artifacts to
+# Github pre-release.
+name: Trunk Artifacts
+
```

Comment:
Please add concurrency group to prevent parallel releases
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true

Reply:
Done